qiskit_nature/second_q/transformers/active_space_transformer.py

```diff
@@ -46,36 +46,46 @@ class ActiveSpaceTransformer(BaseTransformer):
     :math:`F^I_{pq} = h_{pq} + \sum_i 2 g_{iipq} - g_{iqpi}` and the inactive energy which is
     given by :math:`E^I = \sum_j h_{jj} + F ^I_{jj}`, where :math:`i` and :math:`j` iterate over
     the inactive orbitals.
-    By using the inactive Fock operator in place of the one-electron integrals, `h1`, the
+    By using the inactive Fock operator in place of the one-electron integrals, :math:`h_{1}`, the
     description of the active space contains an effective potential generated by the inactive
     electrons. Therefore, this method permits the exclusion of non-core electrons while
     retaining a high-quality description of the system.
 
     For more details on the computation of the inactive Fock operator refer to
     https://arxiv.org/abs/2009.01872.
 
-    The active space can be configured in one of the following ways through the initializer:
-        - when only `num_electrons` and `num_spatial_orbitals` are specified, these integers
-          indicate the number of active electrons and orbitals, respectively. The active space will
-          then be chosen around the Fermi level resulting in a unique choice for any pair of
-          numbers.  Nonetheless, the following criteria must be met:
-
-            #. the remaining number of inactive electrons must be a positive, even number
-
-            #. the number of active orbitals must not exceed the total number of orbitals minus the
-               number of orbitals occupied by the inactive electrons
-
-        - when, `num_electrons` is a tuple, this must indicate the number of alpha- and beta-spin
-          electrons, respectively. The same requirements as listed before must be met.
-        - finally, it is possible to select a custom set of active orbitals via their indices using
-          `active_orbitals`. This allows selecting an active space which is not placed around the
-          Fermi level as described in the first case, above. When using this keyword argument, the
-          following criteria must be met *in addition* to the ones listed above:
-
-            #. the length of `active_orbitals` must be equal to `num_spatial_orbitals`. Note, that
-               we do **not** infer the number of active orbitals from this list of indices!
-
-            #. the sum of electrons present in `active_orbitals` must be equal to `num_electrons`.
+    .. raw:: html
```

Comment:
Can we go back to standard formatting syntax for Sphinx and not embed html. Maybe it looks better but it should be following the standard way so its formatted by Sphinx - if needed we should improve things throughout  as mentioned above.

Reply:
The reason I went with the raw html solution was because this is what it looks like with the sphinx generated styles:

<img width="849" alt="Screenshot 2022-11-01 at 1 38 02 PM" src="https://user-images.githubusercontent.com/23662430/199320732-dc26bae1-0f0b-483e-b90f-fdbb678782de.png">


I wasn't able to find a way to get around this without using raw html. If you would still prefer the sphinx generated style lmk and I'll remove the raw html

Reply:
I think that's because the number list is indented and its treating that whole thing as a definition list. Try outdenting the numbered list, but you will need to add a blank line preceding it (after that text) to ensure its seen correctly.

Reply:
I agree that we should not embed raw html. I also verified that dedenting and adding a blank line between the paragraph and first list item fixes the rendering :+1:

Reply:
Ah THANK YOU i was struggling for ages with indenting and rebuilding but couldnt get the right combo to make it work. I'll give this a try 😄